mesa/agent.py

```diff
@@ -46,6 +47,7 @@ def __init__(self, unique_id: int, model: Model) -> None:
         """
         self.unique_id = unique_id
         self.model = model
+        self.cell: Cell | None = None
```

Comment:
Is it logical it has both a Cell and a Position? Maybe it is, or can one be an property of the other. Mostly to prevent them from going out of sync.

Reply:
No, its kind of reduntant. Pos could be a property that just links to `cell.coordinate`, but that would break exisiting models.